server/bin/pbench-verify-indexed.py

```diff
@@ -19,9 +19,10 @@
 
 _NAME_ = "pbench-verify-indexed"
 
-tb_pat = re.compile(
-    r"\S+_(\d\d\d\d)[._-](\d\d)[._-](\d\d)[T_](\d\d)[._:](\d\d)[._:](\d\d)\.tar\.xz"
-)
+tb_pat = re.compile(pbench.TAR_BALL_NAME_W_TAR_PAT_S)
+
+# Number of JSON documents to fetch in one batch for the scan operation.
+batch_size = 10000
```

Comment:
By normal conventions, that really ought to be `BATCH_SIZE` as a constant. 😁 

Although if we're not already drowning in one-pluses, you could promote this to a CLI parameter as `--batch` with a default value of 1000.